diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index 704c65a..8b8e922 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -13,8 +13,8 @@ android {
         applicationId = "com.fush.erp.recovery"
         minSdk = 26
         targetSdk = 36
-        versionCode = 92
-        versionName = "0.15.4.51.1-ui-language-theme-controls"
+        versionCode = 93
+        versionName = "0.15.4.52-ui-deep-core-localization"
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     }
