--- a/app/build.gradle.kts	2026-08-15 01:31:55.134140766 +0000
+++ b/app/build.gradle.kts	2026-08-15 01:37:20.652377549 +0000
@@ -13,8 +13,8 @@
         applicationId = "com.fush.erp.recovery"
         minSdk = 26
         targetSdk = 36
-        versionCode = 75
-        versionName = "0.15.4.36-ui-purchases-suppliers"
+        versionCode = 76
+        versionName = "0.15.4.37-ui-accounting-treasury"
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     }
