diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index fd94753..71c3309 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -13,8 +13,8 @@ android {
         applicationId = "com.fush.erp.recovery"
         minSdk = 26
         targetSdk = 36
-        versionCode = 90
-        versionName = "0.15.4.50.1-ui-theme-toggle"
+        versionCode = 91
+        versionName = "0.15.4.51-ui-core-operations-localization"
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     }
