diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index bd29ffa..125b04d 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -13,8 +13,8 @@ android {
         applicationId = "com.fush.erp.recovery"
         minSdk = 26
         targetSdk = 36
-        versionCode = 80
-        versionName = "0.15.4.41-ui-assets-maintenance-expenses"
+        versionCode = 81
+        versionName = "0.15.4.42-ui-reports-adaptive"
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     }
